apps/src/code-studio/levels/dialogHelper.js

```diff
@@ -31,7 +32,7 @@ export function showDialog(type, callback, onHidden) {
 
   // Use our prefabricated dialog content.
   var content = document.querySelector("#" + type + "-dialogcontent").cloneNode(true);
-  var dialog = new window.Dialog({
+  var dialog = new Dialog({
```

Comment:
Seeing that `Dialog` used to be on `window`, I expect to see us changing whatever code originally put it there... but I don't see it in this diff.  Are we intentionally still putting Dialog on window?